<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AstroDataWidget</class>
 <widget class="QMainWindow" name="AstroDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Астрономические данные</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="DateIntervalEdit" name="dateIntervalEdit">
        <property name="isHorisontal" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="StationWidget" name="stationWidget">
        <property name="isMapButtonVisible" stdset="0">
         <bool>true</bool>
        </property>
        <property name="isHorisontal" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="wgtSaveLoadBtns" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pbOpen">
           <property name="toolTip">
            <string>Открыть календарь</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/meteo/icons/file_open.png</normaloff>:/meteo/icons/file_open.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSave">
           <property name="toolTip">
            <string>Сохранить календарь</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/meteo/icons/save.png</normaloff>:/meteo/icons/save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbPrint">
           <property name="toolTip">
            <string>Распечатать календарь</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/meteo/icons/print.png</normaloff>:/meteo/icons/print.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spButtons">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pbMakeUp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Сформировать</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Время:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="4">
       <widget class="QComboBox" name="cmbtimespec">
        <item>
         <property name="text">
          <string>UTC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Местное</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tvCalendar">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationWidget</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/stationwidget.h</header>
  </customwidget>
  <customwidget>
   <class>DateIntervalEdit</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/intervaledit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
